src/Connection.cpp

```diff
@@ -220,7 +220,6 @@ removeFromNodes() const
   if (auto n = _outNode.lock())
     n->nodeState().eraseConnection(PortType::Out, _outPortIndex, id());
   
-  updated(*this);
```

Comment:
I don't think updated should go in here because this is always right before it is deleted, not exactly an update